packages/datagrid/__mocks__/reakit.js

```diff
@@ -0,0 +1,6 @@
+jest.mock('reakit', () => {
```

Comment:
A word about this?

Reply:
we need to mock the id generation to have use snapshot testing
most project use `uuid` lib but DS is using reakit helper